contents/handbook/people/side-gigs.md

```diff
@@ -4,32 +4,32 @@ sidebar: Handbook
 
 These side gigs may sometimes earn you money. Sometimes, you may one day want your side gig to become your main gig.
 
-We have deliberately called them "side gigs", as we are ok with you earning money on the side. We are not ok with this being your main focus and PostHog being just a paycheck. Quite simply, we are too small for PostHog not to be your main motivation.
+We have deliberately called them "side gigs", as we are ok with you earning money on the side. We are not ok with this being your main focus and PostHog being just a paycheck. Quite simply, we are too small for PostHog not to be your main motivation. For this reason, we also currently don't offer part time work as an option at PostHog. 
 
 ## Managing time
 
 The key distinction to something being a side gig, and thus it being appropriate, is its impact on your work and the amount of time involved.
 
-A few hours a month on a paid side gig is acceptable. Over 10 hours a week on a paid or open source side gig is a significant amount of extra work which we would expect to start impacting your performance.
+A few hours a month on a paid side gig is acceptable. Over 10 hours a week on a paid or open source side gig is a significant amount of extra work which we would expect to start impacting your performance. In any case, side gigs should by default be something you work on in your personal time. 
 
-If you are doing paid speaking, make it clear that you work for us, and the exception based on time does not apply - we view this as a great way to get PostHog's name out there.
+If you are doing paid speaking, the exception based on time does not apply so long as you make it clear that you work for us - we view this as a great way to get PostHog's name out there.
 
-In a few cases, you may want your side gig to become your full time work one day. That is ok - please just let us know, so we can create a plan. We will try to match you with tasks at PostHog that will help your long term goals, while not impacting your work performance, and will create a timeframe for you that works. We know the key to motivated people is to help you achieve your long term goals, and to align this with what PostHog needs, whether or not you eventually achieve them with us.
+In a few cases, you may want your side gig to become your full time work one day. That is ok - please just let us know, so we can create a plan. We know the key to motivated people is to help you achieve your long term goals, and to align this with what PostHog needs, whether or not you eventually achieve them with us.
 
 Above everything else, if you are going above and beyond for PostHog and you're still able to look after yourself properly, side gigs (whether paid or unpaid) are totally fine. We don't think that's possible beyond a certain level of time/energy commitment to them, but we are very happy for you to spend a little time on them each week.
 
 ## Intellectual property
```

Comment:
Is there something around not directly competing with something PostHog has built or might build in the future? Maybe a "chat with us if you think your side project might fall under this"?

Reply:
Added something on this